Bottom: 7713f6dceda2bc2dc9dbe6e0db2b40708523e6aa
Top:    c306df7bf7a7ca3c143c0093a82565c6943a3620
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 14:10:19 +0100

Refresh of demo-of-exporter-specific-expo

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 32e2fca..5b24567 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -477,13 +477,19 @@ common_draw_helpers_init (HID *hid)
 }
 
 void
-exporter_board (HID * hid, BoxType * region)
+common_export_region (HID *hid, BoxType *region)
 {
   HID *old_gui = gui;
   hidGC savebg = Output.bgGC;
   hidGC savefg = Output.fgGC;
   hidGC savepm = Output.pmGC;
 
+  int i, ngroups, side;
+  int plated;
+  int component, solder;
+  int do_group[MAX_LAYER];     /* This is the list of layer groups we will draw. */
+  int drawn_groups[MAX_LAYER]; /* This is the reverse of the order in which we draw them. */
+
   gui = hid;
   Output.fgGC = gui->make_gc ();
   Output.bgGC = gui->make_gc ();
@@ -494,13 +500,6 @@ exporter_board (HID * hid, BoxType * region)
   hid->set_color (Output.pmGC, "erase");
   hid->set_color (Output.bgGC, "drill");
 
-    int i, ngroups, side;
-  int plated;
-  int component, solder;
-  /* This is the list of layer groups we will draw.  */
-  int do_group[MAX_LAYER];
-  /* This is the reverse of the order in which we draw them.  */
-  int drawn_groups[MAX_LAYER];
 
   PCB->Data->SILKLAYER.Color = PCB->ElementColor;
   PCB->Data->BACKSILKLAYER.Color = PCB->InvisibleObjectsColor;
